pkg/Microsoft.Windows.Compatibility/tests/Directory.Build.props

```diff
@@ -7,7 +7,9 @@
     <!-- Setting NETCoreAppMaximumVersion to a high version so that the sdk doesn't complain if we're restoring/publishing for a higher version than the sdk. -->
     <NETCoreAppMaximumVersion>99.9</NETCoreAppMaximumVersion>
     <PackageConflictPreferredPackages Condition="'$(TargetFramework)' != 'netcoreapp2.0'">Microsoft.Private.CoreFx.NETCoreApp;runtime.$(RuntimeIdentifiers).Microsoft.Private.CoreFx.NETCoreApp;$(PackageConflictPreferredPackages)</PackageConflictPreferredPackages>
-    <DisableImplicitFrameworkReferences Condition="$(TargetFramework.Contains('netcoreapp'))">true</DisableImplicitFrameworkReferences>
-    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
+    <IntermediateOutputPath Condition="'$(RestoreOutputPath)' != ''">$(RestoreOutputPath)</IntermediateOutputPath>
```

Comment:
Can you try instead just setting BaseIntermediateOutputPath to RestorePath and deleting BaseIntermediateOutputPath above?  That should work in place of this and MSBuildProjectExtensionsPath.  Also you can remove PreserveCompilationContext since you're no longer relying on that for publish.